backend/prompt_studio/prompt_studio_registry_v2/prompt_studio_registry_helper.py

```diff
@@ -73,6 +73,12 @@ def frame_spec(tool: CustomTool) -> Spec:
                 "default": False,
                 "description": "Enables SinglePass Extraction",
             },
+            "enable_highlight": {
+                "type": "boolean",
+                "title": "Enable highlight",
+                "default": False,
+                "description": "Enables highlight",
```

Comment:
NIT: Even if this might be an item that we'll eventually remove, I suggest adding a verbose description on what this would do / where it'll be useful. 

Also since this is related to manual review - it doesn't apply to OSS users right? We need to atleast track such necessary splits as a tech debt @Deepak-Kesavan - please add a TODO comment if possible
cc: @hari-kuriakose

Reply:
@chandrasekharan-zipstack This applies to all the settings here. I will add a JIRA ticket as tech debt for this and point this file in the description.

Resolving this for now.